### PR TITLE
Add ScrollBarAutomationPeer

### PR DESCRIPTION
**Pull request by Takoooooo**
## What does the pull request do?
This PR adds ScrollBarAutomationPeer which makes ScrollBar available for use in integration tests. ScrollBar's are not discoverable by Narrator because IsContentElementCore set to false as stated in MS documentation.

Note:
[GetOrientationCore](https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/ScrollBarAutomationPeer.cs#L51-L56) is not implemented in Avalonia like in WPF and i don't really know why we may need it so i decided to not implement it.
[SetValueCore](https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/ScrollBarAutomationPeer.cs#L59-L83) is not implemented also because we dont need to set value in percents, we may just set the value directly.

## Fixed issues
Part of https://github.com/AvaloniaUI/Avalonia/issues/10143

### CODE REVIEW

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0031243-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

File: src/Avalonia.Controls/Automation/Peers/ScrollBarAutomationPeer.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    public class ScrollBarAutomationPeer : RangeBaseAutomationPeer
+    {
+        public ScrollBarAutomationPeer(ScrollBar owner) : base(owner)
+        {
+        }
+
+        override protected string GetClassNameCore()
+        {
+            return "ScrollBar";
```
Comment (robloo) [inline src/Avalonia.Controls/Automation/Peers/ScrollBarAutomationPeer.cs:13]: Should this use `nameof()` ? Magic strings are generally frowned upon and I've noticed this with all the other automation peers as well.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0031324-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0031455-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Review by grokys — APPROVED** [review]

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0031794-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Pull request merged**